src/Components/Search/NewSearch/NewSearchBarInput.tsx

```diff
@@ -168,8 +168,8 @@ const NewSearchBarInput: FC<NewSearchBarInputProps> = ({ relay, viewer }) => {
       query: value,
     })
 
-    clearSearchInput()
-    redirect(option.href)
+    resetValue()
+    window.location.href = option.href
```

Comment:
Why is there a hard page load? Shouldn't this be using our router?

(Also does cmd+click still work with these?)

Reply:
> (Also does cmd+click still work with these?)

Actually no! It doesn't work on current prod implementation as well, but would really nice to have. I'll create a ticket for that

Reply:
Yeah, it can be a separate ticket but basically you just detect if the meta key is pressed in the `event` and just return early if it is.